mantis-server/mantis-server-worker/src/main/java/io/mantisrx/server/worker/WorkerIndexHistory.java

```diff
@@ -22,8 +22,8 @@
 
 public class WorkerIndexHistory {
 
-    final Set<Integer> runningWorkerIndex = new HashSet<Integer>();
-    final Set<Integer> terminalWorkerIndex = new HashSet<Integer>();
+    final Set<Integer> runningWorkerIndex = new HashSet<>();
```

Comment:
Can it be private?

Reply:
Fixed in 0b3695b9fe6e1f0d74fd3b85951c14136a8623b5